go-apps/meep-metrics-engine/server/v2/metrics-engine.go

```diff
@@ -556,22 +558,33 @@ func mePostSeqQuery(w http.ResponseWriter, r *http.Request) {
 		duration = params.Scope.Duration
 		limit = int(params.Scope.Limit)
 	}
-	// Get metrics
+	// Get http metrics
+	params.Fields = append(params.Fields, "msg_type", "src", "dst", "endpoint", "body", "method")
```

Comment:
Use the field constants defined in the metrics package.

Reply:
Updated the code with the field constants for all hardcoded values.